sample/sample.cpp

```diff
@@ -25,10 +31,16 @@ void checkstatus()
 
 int main()
 {
-    std::string account_name = "YOUR_ACCOUNT_NAME";
-    std::string account_key = "YOUR_ACCOUNT_KEY";
-    std::shared_ptr<storage_credential>  cred = std::make_shared<shared_key_credential>(account_name, account_key);
-    std::shared_ptr<storage_account> account = std::make_shared<storage_account>(account_name, cred, true);
+    std::shared_ptr<storage_credential> cred = nullptr;
+    if (!access_token.empty())
+    {
+        cred = std::make_shared<token_credential>(access_token);
+    }
+    else 
+    {
+        cred = std::make_shared<shared_key_credential>(account_name, account_key);
```

Comment:
I don't think this line can pass compiling since `account_name` and `acocunt_key` declaration is deleted.

Reply:
They ended up as statics here https://github.com/Azure/azure-storage-cpplite/pull/33/files#diff-74233336935bd512a507f6c578a73584R11-R16
Let me know if you'd prefer them as local variables inside main though